src/Microsoft.AspNet.OData.Shared/DefaultEdmPatchMethodHandler.cs

```diff
@@ -0,0 +1,165 @@
+﻿//-----------------------------------------------------------------------------
+// <copyright file="DefaultEdmODataAPIHandler.cs" company=".NET Foundation">
+//      Copyright (c) .NET Foundation and Contributors. All rights reserved. 
+//      See License.txt in the project root for license information.
+// </copyright>
+//------------------------------------------------------------------------------
+
+using System;
+using System.Collections.Generic;
+using System.Diagnostics.Contracts;
+using System.Linq;
+using Microsoft.OData.Edm;
+
+namespace Microsoft.AspNet.OData
+{
+    /// <summary>
+    /// This is the default Patch Handler for non CLR type. This calss has default Get, Create and Update
+    /// and will do these actions. This will be used when the original collection to be Patched is provided.
+    /// </summary>
+    internal class DefaultEdmODataAPIHandler : EdmODataAPIHandler
+    {
+        IEdmEntityType entityType;
+        ICollection<IEdmStructuredObject> originalList;
+
+        public DefaultEdmODataAPIHandler(ICollection<IEdmStructuredObject> originalList, IEdmEntityType entityType)
```

Comment:
Document this constructor and it's parameters. What does `originalList` comprise of?